peo/.htaccess

```diff
@@ -0,0 +1,19 @@
+# Attiva il Rewrite Engine
+RewriteEngine On
+
+# Redirige richieste generiche all'ontologia RDF come default
+RewriteRule ^$ https://raw.githubusercontent.com/simonegramegna/peo_ontology/main/peo_ontology.rdf [R=302,L]
```

Comment:
If you leave this redirection first, then all the ones below will not work. I would remove line 5.

Reply:
instead I remove the lines belows beacuse the main ontology is at that link, should this work?

Reply:
It will work, but you won't do content negotiation.
If you remove line 5, content negotiation will work for ttl and rdf/xml